### PR TITLE
Skeleton Animation: Clean up

### PR DESCRIPTION
**Pull request by Mugen87**
This PR performs some minor refactoring in `SkinnedMesh` and `Skeleton`:

#### SkinnedMesh

- Moves the code for bones creation to a new method `.initBones()`. This improves the readability of the corresponding constructor.
- Corrects the signature of `.updateMatrixWorld()`
- Removes some linting errors

#### Skeleton

- Changes the property `.identityMatrix` to a closure variable in `.update()`

### CODE REVIEW

File: src/objects/SkinnedMesh.js [hunk 1]
```diff
@@ -159,21 +179,21 @@ SkinnedMesh.prototype = Object.assign( Object.create( Mesh.prototype ), {
 
 	},
 
-	updateMatrixWorld: function () {
+	updateMatrixWorld: function ( force ) {
 
-		Mesh.prototype.updateMatrixWorld.call( this, true );
```
Comment (mrdoob) [inline src/objects/SkinnedMesh.js]: Hmm, I wonder if there was a reason we were forcing `true` here...

Comment (Mugen87) [inline src/objects/SkinnedMesh.js]: Not 100% sure, but this looks redundant to me. I can see no problems as long as we correctly pass the `force` parameter to `.updateMatrixWorld()`.

Comment (mrdoob) [inline src/objects/SkinnedMesh.js]: 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**